pyproject.toml

```diff
@@ -32,7 +32,7 @@ pkgconfig = "^1.5.5"
 pygit2 = "^1.12.0"
 requests = "^2.26.0"
 lhapdf-management = "^0.3"
-pineappl = "^0.7.2"
+pineappl = "^0.7.4"
```

Comment:
We should remember to replace this with the correct version with all the changes later.

Reply:
so far this should be enough...

Reply:
Indeed, hence why I said later. We can merge this now but should replace that version as soon as there is a new pineappl release, otherwise it is conceptually inconsistent.

Reply:
what version do we need? 0.8? will that trigger further changes?

Reply:
we still need to have a PR merged to have fully working and compatible pineappl version.

Reply:
you mean https://github.com/NNPDF/pineappl/pull/289 ?

Reply:
yes, basically everything is related to that one in one way or another.